### DISCUSSION TITLE
Customize Different Spacings For Different Properties?

### DISCUSSION THREAD

**Discussion by fweth** [tailwindlabs/tailwindcss#9715] [Ideas]
Hello, I just commented [here](https://github.com/tailwindlabs/tailwindcss/issues/1232) but not sure if anybody will read, it's an old thread.  I personally think it's best practice to use `rem` for positive space, like `font-size` or `width`, and `px` for negative space (or white space) like `margin`, `padding`, `gap`, etc.  As far as I can see, if I want to achieve this in Tailwind, I need to add custom values for spacing, but then I'd need a set of identifiers for `rem` values and another set of identifiers for `px` values.  Is it possible to configure Tailwind in a way that e.g. `w-4` resolves to `width: 1rem` but `p-4` resolves to `padding: 16px`?

Here is a good article which explains why `rem` everywhere is not ideal:

https://www.joshwcomeau.com/css/surprising-truth-about-pixels-and-accessibility/

**Comment by wongjn** [comment]
Yes, you can set the `padding` values separately from `spacing`, like:
```js
module.exports = {
  theme: {
    extend: {
      padding: {
        4: '16px',
      },
    },
  },
};
```

**Reply by fweth** [reply]
Thanks!!!
